packages/threatspec/PKGBUILD

```diff
@@ -0,0 +1,29 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=threatspec
+pkgver=0.5.0
+pkgrel=1
+pkgdesc='Project to integrate threat modelling into development process.'
+arch=('any')
+groups=('blackarch' 'blackarch-threat-model')
```

Comment:
We don't have a `blackarch-threat-model` group. Please use the existing ones, in this case `blackarch-defensive` and `blackarch-misc`